core/src/wallet/bitcoin/explorers/api/TransactionsParser.hpp

```diff
@@ -44,6 +44,18 @@ namespace ledger {
                 _objectDepth = 0;
             }
 
+            bool StartObject() {
+                _objectDepth += 1;
+
+                if ((_arrayDepth == 1 || _arrayDepth == 0) && _objectDepth == 1) {
+                    BitcoinLikeBlockchainExplorerTransaction transaction;
+                    _transactions->push_back(transaction);
+                    getTransactionParser().init(&_transactions->back());
```

Comment:
This line seems really opaque to me.

 Can you explain it or comment it ?

Reply:
Done !!